plugins/modules/elasticache.py

```diff
@@ -73,6 +73,10 @@
     type: list
     elements: str
     default: []
+  tags:
```

Comment:
If you're extending the `amazon.aws.tags` documentation fragment, you do not need to document tags anymore. I would suggest to add a note about the version when tags support has been added.